docs/tutorials/azure.md

```diff
@@ -89,27 +124,20 @@ A Service Principal with a minimum access level of contribute to the resource gr
   "password": "password",  <-- aadClientSecret value
   "tenant": "AzureAD Tenant Id"  <-- tenantId value
 }
-...
-
-```
-### Other hosting providers
-If the Kubernetes cluster is not hosted by Azure Container Services and you still want to use Azure DNS, you need to create the secret manually. The secret should contain an object named azure.json with content similar to this:
-```
-{
-  "tenantId": "AzureAD tenant Id",
-  "subscriptionId": "Id",
-  "aadClientId": "Service Principal AppId",
-  "aadClientSecret": "Service Principal Password",
-  "resourceGroup": "MyDnsResourceGroup",
-}
-```
-If [Azure Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview) is enabled for virtual machines, then there is no need to create separate service principal. The contents of `azure.json` should be similar to this:
 ```
+
+#### Azure Managed Service Identity (MSI)
+
+If [Azure Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview) is enabled for virtual machines, then there is no need to create separate service principal.
+
+The contents of `azure.json` should be similar to this:
+
+```json
 {
-  "tenantId": "AzureAD tenant Id",
-  "subscriptionId": "Id",
+  "tenantId": "0f4e348f-8aef-4205-9d60-7a53e4ec3a3f",
```

Comment:
you probably don't want to have (your?) private IDs in here... :smirk:

Reply:
No worries, that's `uuidgen` output. I wanted to have something that looks more like a real ID. But now I think it's better to change it to something that looks more like artifical data, will change it!

Reply:
Fixed!